draft-ietf-rats-architecture.md

```diff
@@ -91,22 +91,16 @@ a remote peer (the "Relying Party") to decide whether to consider that
 Attester a trustworthy peer or not.
 RATS are facilitated by an additional vital party, the Verifier.
 
-The Verifier appraises Evidence via Appraisal Policies and creates
-the Attestation Results to support Relying Parties in their decision
-process.
```

Comment:
Why are these lines deleted?  Recommend keeping them.

Reply:
If this was not redundant content, it seems to be vital to highlight that early on.